packages/jest-mock/src/types.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Just temporary in a separate file. This way I can avoid conflicts / changes with current `mocked()` types. Will move it to `index.ts` later.